### ISSUE TITLE
NVDA fails to read separate list items with the mouse, if its option "Use UIA with Microsoft Edge and other Chromium based browsers when available" is enabled

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#12032]
### Steps to reproduce:
1. Launch NVDA.
2. From NVDA's advanced settings, enable the option "Use UIA with Microsoft Edge and other Chromium based browsers when available" (set it to "Yes").
3. From NVDA's mouse settings, enable "Mouse tracking", set the mouse tracking resolution to "Paragraph", enable "Report role when mouse enters object". I don't know if the latter 2 are important, but that is how they are set on my machine.
4. Open Microsoft Edge and load a page containing a list with a few items.
5. Move the mouse cursor over an item from a list (ordered or unordered one).

### Actual behavior:
NVDA says only "list item" for each list item over which the mouse cursor is located.

### Expected behavior:
NVDA should report not only the role of the object ("list item"), but its text as well.

### System configuration
- Operating system: Windows 10 Pro version 20H2 (build 19042.746), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-21710,11ba984f, installed, in bulgarian.
- Microsoft Edge version: 88.0.705.56.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your computer?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, because the feature that causes this issue is available only in alpha versions.

#### If addons are disabled, is your problem still occuring?
Yes, it is.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, because in this case it is probably not related to this issue.

**Comment by Adriani90** [comment]
@msftedad this seems to impact Edge only when using UIA. Can you please forward this to the development team? Thank you very much.

**Comment by msftedad** [comment]
@Adriani90, We have forwarded this issue to development team. We will provide you with the tracking id once we receive update from them. Thanks

**Comment by msftedad** [comment]
Hi @Adriani90, Issue shared with dependent team, for reference, team have created a ticket "issue [Accessibility 233904](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2FCSS-IRT%2FCSS%2520IRT%2F_workitems%2Fedit%2F233904&data=05%7C02%7Cv-ymounika%40microsoft.com%7C7e3bce31a0af48187d7108dd0d9725f3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638681665033098411%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=xO5rnIiwYzIaZWeBJhHg0tF1%2B%2BPSqqxU3eLtyblJlhE%3D&reserved=0)" tracking purposes, you won’t have access to this link, but this can be used as a reference number for this request.
